### PR TITLE
Time stuff

### PR DESCRIPTION
**Pull request by tranma**

### CODE REVIEW

File: data/sea/21-time.h [hunk 1]
```diff
@@ -30,7 +30,20 @@ static iint_t INLINE itime_to_epoch (itime_t x)
...
+static iint_t INLINE itime_to_epoch_seconds (itime_t x)
+{
+  int64_t y, m, d, h, n, s;
+  itime_to_gregorian (x, &y, &m, &d, &h, &n, &s);
+  y = y - 1600;
+
+  m = (m + 9) % 12;
+  y = y - m/10;
+
+  return s + 60*n + 3600*h + (365*y + y/4 - y/100 + y/400 + (m*306 + 5)/10 + (d - 1)) * 86400;
```
Comment (amosr) [inline data/sea/21-time.h]: wow

Comment (tranma) [inline data/sea/21-time.h]: probably not numerically good or something

File: test/Icicle/Test/Data/Time.hs [hunk 2]
```diff
@@ -36,7 +36,19 @@ prop_time_sea_to_days d
...
+
+prop_time_sea_to_seconds :: Time -> Property
+prop_time_sea_to_seconds d
+  = testIO $ do
+  let epochTime = unsafeTimeOfYMD 1600 03 01
+  let expected  = secondsDifference epochTime d
+
+  runRight $ do
+    library <- readLibrary seaTestables
+    f <- function library "testable_itime_to_epoch_seconds" retInt
```
Comment (amosr) [inline test/Icicle/Test/Data/Time.hs]: looks good

**Comment by amosr** [comment]
👍

File: data/sea/21-time.h (hunk 1 above)
Comment (jystic) [inline data/sea/21-time.h]: yeah, it looks numerical great!

**Comment by jystic** [comment]
🔶 late to the party

**Pull request merged**